charts/retool/templates/deployment_backend.yaml

```diff
@@ -244,6 +248,7 @@ spec:
             port: {{ .Values.service.internalPort }}
           initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
           timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
+          periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
```

Comment:
this was present in values.yaml but not in the template, so I assume just an oversight.